src/vs/workbench/contrib/terminal/common/scripts/shellIntegration-bash.sh

```diff
@@ -91,6 +91,16 @@ if [ -n "${VSCODE_ENV_APPEND:-}" ]; then
 	builtin unset VSCODE_ENV_APPEND
 fi
 
+# Register Python shell activate hooks
+if [ -n "$VSCODE_BASH_ACTIVATE" ] && [ "$TERM_PROGRAM" = "vscode" ]; then
+	# Prevent crashing by negating exit code
+	if ! builtin eval "$VSCODE_BASH_ACTIVATE"; then
+		__vsc_activation_status=$?
+		# TODO: Figure out how to add 'star' icon similar to terminal restored.
```

Comment:
I was hoping I can do the star icon next to the message like the `History restored` message when restoring terminal.

Was thinking something like https://github.com/microsoft/vscode/blob/f4c18a6849f25fa97b6ff7e26f9105152a2ac9e3/src/vs/workbench/contrib/terminal/browser/terminalInstance.ts#L1689 but not sure how to get this on the shell script side🧐

Reply:
For that message you'll need to write the raw escape sequences. Try printing these:

```
\x1b[0m\x1b[7m * \x1b[0;103m <message>\x1b[0m
\x1b[0m\x1b[7m * \x1b[0m <message>
```